tests/test_web_server.py

```diff
@@ -26,13 +26,15 @@
         raise exc
 
     logger = mock.Mock()
-    server = await aiohttp_raw_server(handler, logger=logger)
+    server = await aiohttp_raw_server(handler, logger=logger, debug=False)
     cli = await aiohttp_client(server)
     resp = await cli.get('/path/to')
     assert resp.status == 500
+    assert resp.headers['Content-Type'].startswith('text/plain')
 
     txt = await resp.text()
-    assert "<h1>500 Internal Server Error</h1>" in txt
+    assert txt.startswith("500 Internal Server Error")
```

Comment:
could you please use consistent quotes style?
```suggestion
    assert txt.startswith('500 Internal Server Error')
```

P.S. you can accept such small patch suggestions with right from GitHub UI :)

Reply:
Ok, but this is not related to my pr. The entire file (and project) uses inconsistent quotes.